README.md

```diff
@@ -6,33 +6,32 @@
 [![CI](https://github.com/OpenZeppelin/openzeppelin-monitor/actions/workflows/ci.yaml/badge.svg)](https://github.com/OpenZeppelin/openzeppelin-monitor/actions/workflows/ci.yaml)
 [![Release Workflow](https://github.com/OpenZeppelin/openzeppelin-monitor/actions/workflows/release-please.yml/badge.svg)](https://github.com/OpenZeppelin/openzeppelin-monitor/actions/workflows/release-please.yml)
 
-
-> :warning: This software is in alpha. Use in production environments at your own risk.
-
 In the rapidly evolving world of blockchain technology, effective monitoring is crucial for ensuring security and performance. OpenZeppelin Monitor is a blockchain monitoring service that watches for specific on-chain activities and triggers notifications based on configurable conditions. The service offers multi-chain support with configurable monitoring schedules, flexible trigger conditions, and an extensible architecture for adding new chains.
 
 [Install](https://docs.openzeppelin.com/monitor#getting_started) | [User Docs](https://docs.openzeppelin.com/monitor) | [Quickstart](https://docs.openzeppelin.com/monitor/quickstart) | [Crate Docs](https://docs.openzeppelin.com/monitor/rust_docs/doc/openzeppelin_monitor/)
 
 ## Features
 
-- Multi-chain support
-- Configurable monitoring schedules
-- Flexible trigger conditions
-- Extensible architecture for adding new chains
+- **Real-time Monitoring**: Watch blockchain networks in real-time for specific events and transactions
+- **Smart Filtering**: Use flexible expressions to define exactly what you want to monitor
+- **Multi-notification Support**: Send alerts via Slack, Discord, Email, Telegram, Webhooks, or custom scripts
+- **Configurable Scheduling**: Set custom monitoring schedules using cron expressions
+- **Data Persistence**: Store monitoring data and resume from checkpoints
+- **Extensible Architecture**: Easy to add support for new blockchains and notification types
 
 ## Supported Networks
 
-- EVM
-- Stellar
+- **EVM-Compatible Networks**
```

Comment:
Can we mention list of evm compatible networks that we support or are we saying we support all of them?

Reply:
Theoretically all